addons/account_edi_ubl_cii/models/account_edi_common.py

```diff
@@ -350,7 +350,7 @@ def _import_invoice_ubl_cii(self, invoice, file_data, new=False):
                 if invoice.message_main_attachment_id and \
                         invoice.message_main_attachment_id.name.endswith('.xml') and \
                         'pdf' not in invoice.message_main_attachment_id.mimetype:
-                    invoice.message_main_attachment_id = attachment
+                    invoice._message_set_main_attachment_id(attachment.ids, force=True)
```

Comment:
So we only set if there is already one? Kind of strange, but it was like that before I guess.